### ISSUE TITLE
Any reason why I cannot configure my own embedding function/method for a workspace? (instead of openAI embeddings)

### ISSUE THREAD

**Issue by vpinigin** [Mintplex-Labs/vector-admin#137]
### What would you like to see?

I was wondering why I cannot use another embeding method for uploading docs? Why the only way of doing that is using OpenAI embedding models only?

**Comment by timothycarambat** [comment]
This is because we have not improved VectorAdmin since deciding to build out AnythingLLM. This of course should be allowed but for our first iteration of VA, we just kept the implementation very simple.

Being that embedding models and even db's with built-in embedders are quite popular and common now, this needs to be built out more to suit that use-case.

**Comment by oatmealm** [comment]
Yes, please indicate in the documentation. I wouldn't have tried vector-admin had I known I need an OpenAi account to embed documents. Is there a quick fix to control base url and point at a compatible service endpoint? I.e Ollama or any other service.
Thanks
